src/functions.ts

```diff
@@ -29,3 +31,19 @@ export const addUser = async ({ name, email, password, birthDate }) => {
     birthDate
   });
 };
+
+export const findUserData = async (email: string) => {
+  return await AppDataSource.manager.findOneBy(User, {
+    email
+  });
+};
+
+export const findUserId = async (id: string) => {
+  return await AppDataSource.manager.findOneBy(User, {
+    id
+  });
+};
```

Comment:
```suggestion
export const findUserById = async (id: string) => {
  return await AppDataSource.manager.findOneBy(User, {
    id
  });
};
```

O nome dá a entender que vc vai obter apenas o id de usuário. Mas na vdd vc retorna tudo, e busca pelo id.